### PR TITLE
Keep birth_date a simple string in the User model

### PR DESCRIPTION
**Pull request by omgitsbillryan**
- Removed conversions from `String` to `Time` when parsing `birth_date` from SAML attributes
- Convert the `birth_date` string to desired format when calling MVI

### CODE REVIEW

**Comment by omgitsbillryan** [comment]
Relevant Issue
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/408

File: app/models/user.rb [hunk 1]
```diff
@@ -18,7 +18,7 @@ class User < Common::RedisStore
   attribute :middle_name
   attribute :last_name
   attribute :gender
-  attribute :birth_date, Common::UTCTime
+  attribute :birth_date
```
Comment (carlmjohnson) [inline app/models/user.rb]: What if you split this into two attributes, one that was a string and another that parsed the string with `Date.parse()`? That way, the data would be stored correctly, and downstream users wouldn't have to know about the right way of parsing the string.

Comment (omgitsbillryan) [inline app/models/user.rb]: That's a neat idea.  I think it's better encapsulation.

Comment (omgitsbillryan) [inline app/models/user.rb]: I've coded this up, and having done so, I've changed my mind :-).  

MVI is the only component that needs knowledge of this "_special formatting_".  Putting this formatted birth_date in the User model pollutes an already overpopulated model.

**Comment by kreek** [comment]
LGTM (tested local MVI and it's querying as expected) 👍

File: app/models/user.rb (hunk 1 above)
Comment (saneshark) [inline app/models/user.rb]: Common::UTCTime is an attribute type that I created. You should be able to just create your own coercer and have it format the input however you need.

**Comment by robbiethegeek** [comment]
Let's see if we can land this since the profile is showing this data wrong for the front-end.

**Pull request merged**